### ISSUE TITLE
build failed with CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler

### ISSUE THREAD

**Issue by peyman-hakemi** [Shopify/flash-list#642]
## Current behavior

I am trying to build my project on mac os with yarn ios but I'm getting the below build error. 
everything is ok in android.

![image](https://user-images.githubusercontent.com/38393852/197331629-2209a7a2-1fd8-452c-97a7-e4ee7e99a64e.png)

## To Reproduce
every time I installed this library on the latest react native version I'm getting this error on running on ios.

## I Have Tried
1. Clear de DerivedData folder
rm -rf ~/Library/Developer/Xcode/DerivedData
2. Clean Xcode project
cmd + shift + k
3. Reboot the mac

## Platform:

- [x] iOS

## Environment

@shopify/flash-list : 1.3.1

![image](https://user-images.githubusercontent.com/38393852/197331808-315dcd89-f967-42c9-98f8-115efcb040b4.png)

**Comment by naqvitalha** [comment]
Did you forget running pod install by any chance?

**Comment by peyman-hakemi** [comment]
@naqvitalha  No, I didn't; I even tried clearing my node_module and pods and pod.lock

**Comment by peyman-hakemi** [comment]
@Fannolo Hi, no, I couldn't solve this problem. If you find any solution, I would appreciate it if you let me know

**Comment by cfbadr** [comment]
Any news on this bug ?
